src/main/java/frc/robot/Robot.java

```diff
@@ -91,21 +91,17 @@ public void robotInit() {
         SmartDashboard.putData("ring light on", led.ringLightOn());
         SmartDashboard.putData("ring light off", led.ringLightOff());
 
-        autoChooser.setDefaultOption("Nothing", new InstantCommand());
-        autoChooser.addOption("Pass the Line", drive.drivePastLine());
-        autoChooser.addOption("Shoot 3 Balls and Pass Line", shootAuto());
+        autoChooser.setDefaultOption("Test", drive.autonomousCommand("Test"));
+        autoChooser.addOption("Slolom", drive.autonomousCommand("Slolom"));
+        autoChooser.addOption("Barrel", drive.autonomousCommand("Barrel"));
+        autoChooser.addOption("Bounce", drive.autonomousCommand("Bounce1").andThen(drive.autonomousCommand("Bounce2"),
+                drive.autonomousCommand("Bounce3"), drive.autonomousCommand("Bounce4")));
 
         Shuffleboard.getTab("Shuffleboard").add("Autonomous", autoChooser);
 
         drive.setDefaultCommand(drive.drive(driveController::getTriggers, () -> driveController.getX(Hand.kLeft)));
         indexer.setDefaultCommand(indexer.indexBall());
-
-        // protovision
-        camera0 = CameraServer.getInstance().startAutomaticCapture(0);
-        camera0.setResolution(320, 240);
-        camera0.setFPS(20);
-        // videoSink.getProperty("compression").set(70);
-        videoSink = CameraServer.getInstance().getServer();
+        DriverStation.getInstance().silenceJoystickConnectionWarning(true);
```

Comment:
Are we not using camera streaming at all with this branch?

Reply:
nopee we're using the realsense now